src/components/sidebar/profile/profile.js

```diff
@@ -9,15 +9,10 @@ import useStyles from "./styles";
 import { Avatar, Typography } from "@material-ui/core";
 
 const Profile = props => {
-  const { className, ...rest } = props;
+  const { className, user, ...rest } = props;
```

Comment:
Why not place the `{ className, user, ...rest }` in the function parameters, once `props` is not used in the component?